### ISSUE TITLE
Bluehost Plugin v3.5.0 breaks Woocommerce store

### ISSUE THREAD

**Issue by achmed13** [bluehost/bluehost-wordpress-plugin#770]
**Summarize the issue (briefly)**
After updating the Bluehost plugin to v3.5.0, my Woocommerce store no longer displays unless I am logged in to Wordpress. I get an error message on the screen with a blue ice cream cone image. After rolling back to Bluehost v3.4.2 it works again.

* Bluehost Plugin Version: 3.5
* WooCommerce Version 8.2.1
* Wordpress Version 6.3.2

Here is the error in the Woocommerce logs:
```
2023-10-27T16:24:36+00:00 CRITICAL Uncaught Error: Class "NewfoldLabs\WP\Module\Onboarding\WP_ADMIN" not found in /home3/oudolfga/public_html/wp-content/plugins/bluehost-wordpress-plugin/vendor/newfold-labs/wp-module-onboarding/includes/Services/PluginService.php:131
Stack trace:
#0 /home3/oudolfga/public_html/wp-content/plugins/bluehost-wordpress-plugin/vendor/newfold-labs/wp-module-onboarding/includes/Application.php(76): NewfoldLabs\WP\Module\Onboarding\Services\PluginService::configure_activation_transient()
#1 /home3/oudolfga/public_html/wp-content/plugins/bluehost-wordpress-plugin/vendor/newfold-labs/wp-module-onboarding/bootstrap.php(51): NewfoldLabs\WP\Module\Onboarding\Application->__construct(Object(NewfoldLabs\WP\ModuleLoader\Container))
#2 [internal function]: {closure}(Object(NewfoldLabs\WP\ModuleLoader\Container), Object(NewfoldLabs\WP\ModuleLoader\Module))
#3 /home3/oudolfga/public_html/wp-content/plugins/bluehost-wordpress-plugin/vendor/newfold-labs/wp-module-loader/includes/functions.php(82): call_user_func(Object(Closure), Object(NewfoldLabs\WP\ModuleLoader\Container), Object(NewfoldLabs\WP\ModuleLoader\Module))
#4 /home3/oudolfga/public_html/wp-includes/class-wp-hook.php(310): NewfoldLabs\WP\ModuleLoader\load('')
#5 /home3/oudolfga/public_html/wp-includes/class-wp-hook.php(334): WP_Hook->apply_filters(NULL, Array)
#6 /home3/oudolfga/public_html/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#7 /home3/oudolfga/public_html/wp-settings.php(610): do_action('after_setup_the...')
#8 /home3/oudolfga/public_html/wp-config.php(158): require_once('/home3/oudolfga...')
#9 /home3/oudolfga/public_html/wp-load.php(50): require_once('/home3/oudolfga...')
#10 /home3/oudolfga/public_html/wp-blog-header.php(13): require_once('/home3/oudolfga...')
#11 /home3/oudolfga/public_html/index.php(17): require('/home3/oudolfga...')
#12 {main}
  thrown in /home3/oudolfga/public_html/wp-content/plugins/bluehost-wordpress-plugin/vendor/newfold-labs/wp-module-onboarding/includes/Services/PluginService.php on line 131
```
![Screenshot 2023-10-27 123559](https://github.com/bluehost/bluehost-wordpress-plugin/assets/3248232/a947aa8e-4d0c-4b2d-a759-b7414b78fb24)

**Comment by circlecube** [comment]
Thanks for the heads up! We noticed this issue as well and have a fix coming for it in the next release. It's already merged too: #780

**Comment by circlecube** [comment]
This was fixed with the 3.5.2 release. Thanks again for reporting the issue.
